### PR TITLE
fixed currentVaccineNumber lapping on top of each other

### PR DESCRIPTION
**Pull request by godwin-morwin**

### CODE REVIEW

**Comment by FlavioAdamo** [comment]
When you use a default `width` this happens:
image
The green background in `.currentVaccineNumber` should be the same size as the number.
Can you work on that?

**Comment by godwin-morwin** [comment]
@FlavioAdamo sure , i got it

**Comment by godwin-morwin** [comment]
Hi @FlavioAdamo ,
Are you ok with this solution ?

covidtrcker

**Comment by FlavioAdamo** [comment]
### The `.currentVaccineNumber` text is not aligned with `.differenceounter` 
image
it should be like this:
image

### The syringe.png is not aligned to the center:
image
Should be like this:
image

Everyone make mistake, if you can't handle this issues is really not a problem, we have a lot of issues open that you can look at.
But if you want to try again(i recommend you this) we have no rush here, take your time.

**Comment by godwin-morwin** [comment]
@FlavioAdamo ok let me try again

**Comment by FlavioAdamo** [comment]
@Godwindaniel10 Take your time, Even if it takes you a couple of days it's not a problem, the important thing is to it right.

**Comment by godwin-morwin** [comment]
@FlavioAdamo Thanks, truthfully i was rushing it and cant think strait on the issue. But now i got time to get it fixed
